homeassistant/components/enocean/binary_sensor.py

```diff
@@ -36,7 +37,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     add_entities([EnOceanBinarySensor(dev_id, dev_name, device_class)])
```

Comment:
Each platform needs to implement `async_setup_entry` to add its entities. The old `setup_platform` should not be used for this anymore.

Reply:
I can definitely start working on the platforms also in this PR.
How do I access the yaml configuration from async_setup_entry ?
I have looked at other components configured this way (rfxtrx, tcp, template, sql), but no luck, they all rely on setup_platform.

Reply:
We can't access the platform config yaml. These will go away. We'll have access to the config entry and its data.

The integration can only import config yaml to a config entry as we already do in `async_setup`.

Reply:
I just pushed a first proposal for the import of the entities found in the YAML configuration. Does that match the expected way to do imports ?